addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -215,42 +183,33 @@ class DescriptionScreen extends Component {
                 this.labelToId[r.label] = r.id;
             });
         } else {
-            labels = [_t("No result found, broaden your search.")];
+            labels = [_t(request.term)];
```

Comment:
Just to be sure, this `_t()` was about translating its input right?

Reply:
It will extract the given term to the `.pot` file which then will generate traduction requests on transifex website which then will generate the according `.po` file with translation in Odoo.
But here obviously it won't do anything as there is nothing to be sent on transifex for translation.

Reply:
Don't hesitate to ask whenever you find something that you don't understand or have doubt about, even if it's 10 times a day.
Here you modified that line and should have been curious about it.
Ask and get answer, there is no harm. We are here for that and just that: spread knowledge. It will increase your learning curve x10 ;-)